frontend/src/features/acquisition/list/AcquisitionSearchResults/columns.tsx

```diff
@@ -20,11 +19,9 @@ export const columns: ColumnWithProps<Api_AcquisitionFile>[] = [
     Cell: (props: CellProps<Api_AcquisitionFile>) => {
       const { hasClaim } = useKeycloakWrapper();
       if (hasClaim(Claims.ACQUISITION_VIEW)) {
-        // TODO: PSP-4400 Remove icon when file number field is added to CREATE ACQUISITION FILE form.
-        // The icon is here so we can open ACQ File details from list
         return (
           <Link to={`/mapview/sidebar/acquisition/${props.row.original.id}`}>
-            {props.row.original.fileNumber} <FaExternalLinkAlt size="2rem" />
+            {props.row.original.fileNumber}
           </Link>
```

Comment:
We can now remove that temporary icon because File Number is now a required field in the database and guaranteed to have a value